packages/calypso-e2e/src/media-helper.ts

```diff
@@ -117,15 +117,27 @@ export function deleteFile( filePath: string ): void {
  * @param {string} param0.sourceFileName Basename of the source file to be cloned.
  * @returns {string} Full path to the generated test file.
  */
-export function createTestFile( { sourceFileName }: { sourceFileName: string } ): string {
-	const testFileName = `${ createTimestamp() }.${ sourceFileName.split( '.' ).pop() }`;
+export function createTestFile( {
+	sourceFileName,
+	testFileName,
+}: {
+	sourceFileName: string;
+	testFileName?: string;
+} ): string {
+	// If the output `testFileName` is defined, use that as part of the final filename.
+	let fileName =
+		testFileName === undefined ? createTimestamp() : `${ createTimestamp() }-${ testFileName }`;
```

Comment:
nit for dropping the ternary:
```suggestion
	let fileName = createTimestamp();
	if ( testFileName ) {
		fileName += `-${ testFileName }`;
	}
```

Reply:
Replaced the ternary operator. Thanks!